etl/steps/export/s3/co2_data/latest/owid_co2.py

```diff
@@ -0,0 +1,125 @@
+"""Garden step that combines various datasets related to greenhouse emissions and produces the OWID CO2 dataset.
+
+The combined datasets are:
+* Global Carbon Budget - Global Carbon Project.
+* National contributions to climate change - Jones et al.
+* Greenhouse gas emissions by sector - Climate Watch.
+* Primary energy consumption - EI & EIA.
+
+Additionally, OWID's regions dataset, population dataset and Maddison Project Database (Bolt and van Zanden, 2023) on
+GDP are included.
+
+Outputs:
+* The data in three different formats will also be uploaded to S3, and will be made publicly available, in:
+  * https://nyc3.digitaloceanspaces.com/owid-public/data/co2/owid-co2-data.csv
+  * https://nyc3.digitaloceanspaces.com/owid-public/data/co2/owid-co2-data.xlsx
+  * https://nyc3.digitaloceanspaces.com/owid-public/data/co2/owid-co2-data.json
+
+"""
+import json
+import tempfile
+from pathlib import Path
+
+import pandas as pd
+from owid.catalog import Table
+from owid.datautils.s3 import S3
+from structlog import get_logger
+from tqdm.auto import tqdm
+
+from etl.helpers import PathFinder
+
+# Initialize logger.
+log = get_logger()
+
+# Define S3 base URL.
+S3_URL = "https://nyc3.digitaloceanspaces.com"
```

Comment:
Our preferred S3 storage is now R2 on Cloudflare, though we don't have a `owid-public` bucket there yet. I could set it up there & add proper permissions if we wanted to.

Although on second though, since we have all our stuff in DigitalOcean, it'd make more sense to migrate everything at once at some point in the future.

Reply:
I leave this decision up to you. Please let me know if you want me to change anything, or feel free to change it yourself, thanks.